aerogear/mobile-security-service-operator
# fix(monitor): service has been not created for the operator

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9736

## What
- Change order for the operator service be created
- Remove the operator_service.yaml.yaml since it should not be used. 

## Why
It is required in the monitoring feature.

## Verification Steps
- Install the project with : `image: cmacedo/mobile-security-service-operator:suggestion`
- after check that the service `service/mobile-security-service-operator ` was created. 

 ```shell
oc get all
NAME                                                    READY     STATUS    RESTARTS   AGE
pod/mobile-security-service-797668569b-th57n            1/2       Running   0          8s
pod/mobile-security-service-operator-67c46d6f47-zrgvq   1/1       Running   0          23s

NAME                                          TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
service/mobile-security-service-application   ClusterIP   172.30.216.62    <none>        3000/TCP   9s
service/mobile-security-service-operator      ClusterIP   172.30.70.9      <none>        8383/TCP   19s
service/mobile-security-service-proxy         ClusterIP   172.30.208.159   <none>        80/TCP     9s

NAME                                               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/mobile-security-service            1         1         1            0           9s
deployment.apps/mobile-security-service-operator   1         1         1            1           23s

NAME                                                          DESIRED   CURRENT   READY     AGE
replicaset.apps/mobile-security-service-797668569b            1         1         0         9s
replicaset.apps/mobile-security-service-operator-67c46d6f47   1         1         1         23s

NAME                             HOST/PORT                                            PATH      SERVICES                        PORT      TERMINATION   WILDCARD
route.route.openshift.io/route   route-mobile-security-service.192.168.64.30.nip.io             mobile-security-service-proxy   web       edge          None
```
- Check here that the Grafana worked without the operator_service.yaml be applied.

## Checklist:


## Progress


## Additional Notes
